x/ibc_query/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,80 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	testkeeper "github.com/cosmos/interchain-queries/testutil/keeper"
+	ibcquerykeeper "github.com/cosmos/interchain-queries/x/ibc_query/keeper"
+	"github.com/cosmos/interchain-queries/x/ibc_query/types"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCrossChainQueryResult validates CrossChainQueryResult against the spec.
+// Retrieve the query result from the private store using the query's identifier.
+func TestCrossChainQueryResult(t *testing.T) {
```

Comment:
we should know what this code is testing. That means, code should show us what input and output is.

In this code, input(queryResult) is sent in helper function.
So developer can't know what is input if only see test code.

I think it will be better if testCase struct has explicit input and expect output